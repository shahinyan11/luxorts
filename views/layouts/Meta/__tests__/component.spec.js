import { shallow } from 'enzyme';

import MetaComponent from '../component';

describe('Meta component', () => {
  const component = shallow(<MetaComponent />);

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
