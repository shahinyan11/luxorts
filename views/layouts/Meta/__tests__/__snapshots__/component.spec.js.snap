// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meta component matches snapshot 1`] = `
<Head>
  <meta
    content="width=device-width, initial-scale=1, shrink-to-fit=no maximum-scale=1.0 user-scalable=no"
    name="viewport"
  />
  <meta
    content="ie=edge"
    httpEquiv="x-ua-compatible"
  />
  <meta
    charSet="utf-8"
  />
</Head>
`;
